### ISSUE TITLE
Uncaught TypeError: Cannot read property 'Viewer' of undefined

### ISSUE THREAD

**Issue by ccmostmosthandsome** [Flowm/satvis#40]
node 11.11.0/10.0.0/10.10.0
npm 6.7.0 /5.6.0/
cnpm install 
npm install 
all cant start item

error: 
CesiumController.js:25 Uncaught TypeError: Cannot read property 'Viewer' of undefined
    at new CesiumController (CesiumController.js:25)
    at Object.install (VueCesiumController.js:6)
    at Function.Vue.use (vue.esm.js:5100)
    at eval (Cesium.vue?C:/Users/123/Desktop/orbforce/node_modules/babel-loader/lib??ref--0!C:/Users/123/Desktop/orbforce/node_modules/vue-loader/lib??vue-loader-options:239)
    at Module../node_modules/babel-loader/lib/index.js?!./node_modules/vue-loader/lib/index.js?!./src/components/Cesium.vue?vue&type=script&lang=js& (app.3d870e7c.js?__WB_REVISION__=3d870e7c14472ee0fd3e:862)
    at __webpack_require__ (app.3d870e7c.js?__WB_REVISION__=3d870e7c14472ee0fd3e:80)
    at eval (VM1282 Cesium.vue:2)
    at Module../src/components/Cesium.vue?vue&type=script&lang=js& (app.3d870e7c.js?__WB_REVISION__=3d870e7c14472ee0fd3e:2395)
    at __webpack_require__ (app.3d870e7c.js?__WB_REVISION__=3d870e7c14472ee0fd3e:80)
    at eval (VM1282 Cesium.vue:3)

__webpack_require__.r(__webpack_exports__);
/* harmony export (binding) */ __webpack_require__.d(__webpack_exports__, "CesiumController", function() { return CesiumController; });
/* harmony import */ var Cesium__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! Cesium */ "Cesium");
/* harmony import */ var Cesium__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(Cesium__WEBPACK_IMPORTED_MODULE_0__);
/* harmony import */ var dayjs__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! dayjs */ "./node_modules/dayjs/dayjs.min.js");
/* harmony import */ var dayjs__WEBPACK_IMPORTED_MODULE_1___default = /*#__PURE__*/__webpack_require__.n(dayjs__WEBPACK_IMPORTED_MODULE_1__);
/* harmony import */ var dayjs_plugin_utc__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/*! dayjs/plugin/utc */ "./node_modules/dayjs/plugin/utc.js");
/* harmony import */ var dayjs_plugin_utc__WEBPACK_IMPORTED_MODULE_2___default = /*#__PURE__*/__webpack_require__.n(dayjs_plugin_utc__WEBPACK_IMPORTED_MODULE_2__);
/* harmony import */ var _util_DeviceDetect__WEBPACK_IMPORTED_MODULE_3__ = __webpack_require__(/*! ./util/DeviceDetect */ "./src/modules/util/DeviceDetect.js");
/* harmony import */ var _SatelliteManager__WEBPACK_IMPORTED_MODULE_4__ = __webpack_require__(/*! ./SatelliteManager */ "./src/modules/SatelliteManager.js");
// Import webpack externals

code from CesiumController.js when builded:
dayjs__WEBPACK_IMPORTED_MODULE_1___default.a.extend(dayjs_plugin_utc__WEBPACK_IMPORTED_MODULE_2___default.a);
class CesiumController {
  constructor() {
    this.minimalUI = _util_DeviceDetect__WEBPACK_IMPORTED_MODULE_3__["DeviceDetect"].inIframe() || _util_DeviceDetect__WEBPACK_IMPORTED_MODULE_3__["DeviceDetect"].isIos(); //检测设备

    this.minimalUIAtStartup = _util_DeviceDetect__WEBPACK_IMPORTED_MODULE_3__["DeviceDetect"].inIframe(); //检测设备
    // 声明Cesium地方，作为其本身的属性

    this.viewer = new Cesium__WEBPACK_IMPORTED_MODULE_0___default.a.Viewer("cesiumContainer",  (please attention)

if you also find the problem, can y tell me if its solved ! thank you!

**Comment by ccmostmosthandsome** [comment]
Error will not occur when i remove  import Cesium from "Cesium";  line in cesiumController.js,
why is that?

**Comment by Flowm** [comment]
Ran into the same issue when upgrading from Cesium 1.61.0 to 1.65.0. Likeley caused by their switch to ES6 modules in [Cesium 1.63.0](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md#163---2019-11-01).

Instead of loading Cesium as a separate script, it is now built with webpack which resolved the error for me. Unfortunately CesiumSensorVolumes does not work yet with the new structure yet. I disabled it until I have more time to look into this.
